src/parser/monk/shared/modules/covenants/FallenOrder.tsx

```diff
@@ -0,0 +1,144 @@
+import Analyzer, { SELECTED_PLAYER, SELECTED_PLAYER_PET } from 'parser/core/Analyzer';
+import Abilities from 'parser/core/modules/Abilities';
+import SPELLS from 'common/SPELLS';
+import Events, { DamageEvent, HealEvent, SummonEvent } from 'parser/core/Events';
+import Statistic from 'interface/statistics/Statistic';
+import STATISTIC_ORDER from 'interface/others/STATISTIC_ORDER';
+import STATISTIC_CATEGORY from 'interface/others/STATISTIC_CATEGORY';
+import ItemDamageDone from 'interface/ItemDamageDone';
+import ItemHealingDone from 'interface/ItemHealingDone';
+import React from 'react';
+import BoringSpellValueText from 'interface/statistics/components/BoringSpellValueText';
+import { formatNumber } from 'common/format';
+import SpellUsable from 'parser/shared/modules/SpellUsable';
+import COVENANTS from 'game/shadowlands/COVENANTS';
+
+class FallenOrder extends Analyzer {
+  static dependencies = {
+    spellUsable: SpellUsable,
+    abilities: Abilities,
+  };
+
+  windDamage: number = 0;
+  brewDamage: number = 0;
+  mistDamage: number = 0;
+
+  mistHealing: number = 0;
+  mistOverhealing: number = 0;
+  cloneMap: Map<number, number> = new Map<number, number>();
+
+
+  constructor(options: any) {
+    super(options);
+
+    // In my testing this line always returns false but its what putput uses so I have faith
+    // it will work in the future. (might just be a bad log too)
+    this.active = this.selectedCombatant.hasCovenant(COVENANTS.VENTHYR.id);
+
+    if (!this.active) {
+      return;
+    }
+
+    options.abilities.add({
```

Comment:
rather than using options.abilities, use `this.abilities` (after adding `protected abilities!: Abilities;` to L21)